xep-0156.xml

```diff
@@ -168,6 +174,7 @@ _xmppconnect IN TXT "_xmpp-client-websocket=wss://web.example.com:443/ws"
       <li>HTTP queries for host-meta information MUST be used only as a fallback after the methods specified in <cite>RFC 6120</cite> have been exhausted.</li>
       <li>A domain SHOULD NOT present information in host-meta link records that is available via the DNS SRV records defined in <cite>RFC 6120</cite>.</li>
       <li>The order of XMPP related link entries in the host-meta file SHOULD NOT be interpreted as significant by the presenting domain or the receiving entity.</li>
+      <li>Support for the XML version of the host-meta file is REQUIRED as specified by <cite>RFC 6120</cite> &sect;3 while the JSON version (and other representations) are OPTIONAL.</li>
```

Comment:
Done. Moved it out of the business logic too since I'm not sure it made sense there the way I'd worded it.